docs/guides/creatingasplitter.md

```diff
@@ -25,17 +35,20 @@ spec:
         - image: docker.io/n3wscott/sockeye:v0.7.0@sha256:e603d8494eeacce966e57f8f508e4c4f6bebc71d095e3f5a0a1abaf42c5f0e48
 ```
 
-Open its web interface in browser:
+```
+kubectl apply -f sockeye.yaml
+```
+
+Open the web interface in a browser:
 
 ```shell
 browse $(kubectl get ksvc sockeye -o=jsonpath='{.status.url}')
```

Comment:
Not in the change set I know, but as already mentioned, this is Mac specific, and I'm sure people know what to do with a URL.

Reply:
I removed the browse